src/client/components/ChannelInformation/ChannelInnformation.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const RenderChannelInformation = () => {
+  return <h3>Here is channel info</h3>;
+};
```

Comment:
I think it would be great to have the text as a `prop`, so we can set the text when we want to.

I would do it like this:
```javascript
export const RenderChannelInformation = ({ headline = "Here is channel info" }) => {
  return <h3>{headline}</h3>;
};
```